CRM/Custom/Form/CustomDataTrait.php

```diff
@@ -85,19 +85,14 @@ protected function addCustomDataFieldsToForm(string $entity, array $filters = []
     }
 
     $formValues = [];
+    $customGroupStyle = [];
     foreach ($fields as $field) {
       // Filter for fields in "Inline" style custom groups only, as others are not added to forms via Ajax. Not removing
       // them here would register them in the QuickForm but not have POST values for them, which might result in data
       // loss (most notably radio fields, for which no POST value will be interpreted as a reset).
       // See https://lab.civicrm.org/dev/core/-/issues/5613
-      if (!isset(Civi::$statics[__CLASS__]['customGroups'][$field['custom_group']])) {
-        Civi::$statics[__CLASS__]['customGroups'][$field['custom_group']] = \Civi\Api4\CustomGroup::get(FALSE)
-          ->addSelect('style')
-          ->addWhere('name', '=', $field['custom_group'])
-          ->execute()
-          ->single();
-      }
-      if ('Inline' !== Civi::$statics[__CLASS__]['customGroups'][$field['custom_group']]['style']) {
+      $customGroupStyle[$field['custom_group']] ??= CRM_Core_BAO_CustomGroup::getGroup(['name' => $field['custom_group']])['style'];
```

Comment:
So `$field['custom_group']` was not present when I tested on 6.0 - hence 6.0 break when editing address custom data